objects/task5.js

```diff
@@ -0,0 +1,13 @@
+const users = [
+    {name : "John", email : "john@gmail.com", age : 25},
+    {name : "Stefan", email : "stefan@gmail.com", age : 30},
+    {name : "Daniel", email : "daniel@gmail.com", age : 32}
+]
+
+for (const newUser of users) {
+    const { name, email, age } = users;
```

Comment:
Apply destructuring to newUser but not to users array.

Reply:
By the way, why you name it 'newUser'? It makes me confused. Loop does not create new objects , it just makes iteration. Just 'user' would be better for this variable.

Reply:
Solved all remarks.